### PR TITLE
[RB-45] Add more eslint rules

### PR DESCRIPTION
**Pull request by liuba83**
## Description
Add more eslint rules.

## Changes
Enable more eslint rules in `.eslintrc.json` file. Update all files so that they follow new rules.

### CODE REVIEW

**Review by in43sh — APPROVED** [review]
Overall, looks good! Except the two times when eslint were disabled but that can be fixed later.

File: .eslintrc.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "extends": [
     "eslint:recommended",
     "plugin:react/recommended",
-    "plugin:react-hooks/recommended",
```
Comment (VadimDmitr) [inline .eslintrc.json:9]: Was removing `plugin:react-hooks/recommended` on purpose? Might it cause problems, if we use hooks?

**Review by VadimDmitr — COMMENTED** [review]
The changes look good and improve the code. Only question is about the hooks plugin.

File: .eslintrc.json (hunk 1 above)
Comment (EvgeniiRychkov) [reply .eslintrc.json:9]: I think you should return it.

File: src/pages/About.jsx [hunk 2]
```diff
@@ -33,9 +32,11 @@ const About = () => {
         Meet our team
       </h2>
 
+      {/* eslint-disable jsx-a11y/aria-role */}
```
Comment (EvgeniiRychkov) [inline src/pages/About.jsx]: Remove these lint comments and rename 'role' to something else like 'group'

File: .eslintrc.json (hunk 1 above)
Comment (liuba83) [reply .eslintrc.json:9]: Thank you, I returned it.

File: src/pages/About.jsx (hunk 2 above)
Comment (liuba83) [reply src/pages/About.jsx]: That makes sense, thank you.

**Review by in43sh — APPROVED** [review]

**Review by EvgeniiRychkov — APPROVED** [review]

**Pull request merged**